kedro/framework/project/__init__.py

```diff
@@ -259,3 +261,58 @@ def validate_settings():
         )
 
     importlib.import_module(f"{PACKAGE_NAME}.settings")
+
+
+def find_pipelines() -> Dict[str, Pipeline]:
```

Comment:
Nit: could you add some white lines between some of the steps in the function, it's quite hard to read what parts belong to which if/else/try/catch blocks, e.g. between lines 306 and 307 https://github.com/kedro-org/kedro/blob/9fdd4e868444fd2f75dae19b4db1651862f88db2/kedro/framework/project/__init__.py#L306-L307

Reply:
Done!